examples/modules/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ add_executable(example_module
 )
 
 target_link_libraries(example_module
+  PRIVATE Threads::Threads
```

Comment:
(driveby) make sure we link threads properly. this was necessary to fix builds on linux.